build.gradle

```diff
@@ -71,12 +71,16 @@ static def checkForUpdate() {
         String availableBuildScript = new URL("https://raw.githubusercontent.com/SinTh0r4s/ExampleMod1.7.10/main/build.gradle").newInputStream(parameters).getText()
         String availableBuildScriptHash = getVersionHash(availableBuildScript)
         
-        if (currentBuildScriptHash.empty || availableBuildScriptHash.empty || currentBuildScriptHash == availableBuildScriptHash) return
 
-        println("Build script update available! Check https://github.com/SinTh0r4s/ExampleMod1.7.10/blob/main/build.gradle")
+        if (currentBuildScriptHash.empty || availableBuildScriptHash.empty || currentBuildScriptHash == availableBuildScriptHash)
+            return false
+
+        return true
```

Comment:
What do you think about renaming this to `isUpToDate()` and replacing if with returns to:
```
return currentBuildScriptHash.empty || availableBuildScriptHash.empty || currentBuildScriptHash == availableBuildScriptHash`
```